pkg/apps/blackduck/latest/creater.go

```diff
@@ -238,7 +238,7 @@ func (hc *Creater) initPostgres(bdspec *blackduckapi.BlackduckSpec) error {
 
 	for dbInitTry := 0; dbInitTry < math.MaxInt32; dbInitTry++ {
 		// get the secret from the default operator namespace, then copy it into the hub namespace.
-		adminPassword, userPassword, postgresPassword, err = bdutils.GetDefaultPasswords(hc.KubeClient, hc.Config.Namespace)
+		adminPassword, userPassword, postgresPassword, _, err = util.GetDefaultPasswords(hc.KubeClient, hc.Config.Namespace)
```

Comment:
Probably you need to fix the merge conflict of this PR with Yash changes related to password